tensorboard/webapp/metrics/views/card_renderer/scalar_card_component.ts

```diff
@@ -253,4 +254,12 @@ export class ScalarCardComponent<Downloader> {
 
     return dataTableData;
   }
+
+  onFobSelectTimeChanged(newSelectedTime: LinkedTime) {
+    this.internalSelectedTime = newSelectedTime;
```

Comment:
This is interesting. So the internalSelectedTime updates when updating linked time...but only on the card that is actually being dragged? Was that intentional? 

This could probably lead to some strange behavior. I think this might be a good design decision but, it is hard to tell without playing around with it.

I just wanted to make sure this was intentional and I am ok with it.

Reply:
Here's what I think: dragging the fob will always updates internal selected time. Only when it is "linked" we dispatch timeSelectionChange and update linkedTime in ngrx state to also update fob/data table in other cards.
This is intentional to keep the internal selected time up to date when we uncheck "LinkedTime" in setting panel. Without this there will be a jump because mismatch between internal selected time and linked selected time.

I think the input type "LinkedTime" is where causes the confusing, if we think about it as "a pair of start step and end step", would that be less confusing? (that's also why I change rename "newLinkedTime" to "newSelectedTime" but change the name of type involves a lot of other things)